Assets/Mirror/Runtime/NetworkReaderPool.cs

```diff
@@ -25,6 +25,8 @@ public void Dispose()
     /// </summary>
     public static class NetworkReaderPool
     {
+        static readonly ILogger logger = LogFactory.GetLogger(typeof(NetworkReaderPool));
```

Comment:
set the default log level to error, since previously the log was only enabled when LogFilter.Debug is true

Reply:
not sure what you mean by this as theres only one normal logwarning in that class.

Reply:
for exmaple in weaver tests we have 
```csharp
public static readonly ILogger logger = LogFactory.GetLogger<WeaverTests>(LogType.Exception);
```
Since by default we don't want any logs from the tests.

In this case (and WriterPool) we don't want those warnings by default, so we can set the log level to error so they arn't logged unless people want them to be.